staging/src/k8s.io/client-go/tools/cache/fifo.go

```diff
@@ -215,6 +236,10 @@ func (f *FIFO) Delete(obj interface{}) error {
 	f.lock.Lock()
 	defer f.lock.Unlock()
 	f.populated = true
+	// delete key in `queue`
+	order := f.items[id].(OrderItem).Order
```

Comment:
add the check?

 if o, ok := f.items[id].(OrderItem); ok { .. }

Reply:
ok, thanks for your reply. there is a design problem in my impl, I will fix this in next commit. thanks.